src/views/partials/profile_jobDetails.hbs

```diff
@@ -12,15 +12,15 @@
     <label for="job_search_status">Job Search Status *
       <select id="job_search_status" name="job_search_status">
         <option value="green">Currently Looking</option>
-        <option value="yellow">Maybe interested</option>
+        <option value="orange">Maybe interested</option>
         <option value="red">Not interested</option>
       </select>
     </label>
     <label for="years_experience">Years experience
       <select id="years_experience" name="years_experience">
-        <option value="{min_years_exp: 0, max_years_exp: 2}">Less than two years</option>
-        <option value="{min_years_exp: 2, max_years_exp: 5}">Two to five years</option>
-        <option value="{min_years_exp: 5, max_years_exp: 100}">More than five years</option>
+        <option value="{'min_years_exp': 0, 'max_years_exp': 2}">Less than two years</option>
```

Comment:
This seems overly complicated for what we need.  I think a simpler format below would be better. This would mean updated the DB_build.sql as well. 

Years of Expereince: 
Options: 
Less than 1 year
1-2 years
2-3 years
etc... 

This way hiring manager can search for min years of experience. I don't think weed need a max_years experience.